src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/CmsSigner.cs

```diff
@@ -14,8 +14,6 @@ namespace System.Security.Cryptography.Pkcs
 {
     public sealed class CmsSigner
     {
-        private static readonly Oid s_defaultAlgorithm = Oid.FromOidValue(Oids.Sha256, OidGroup.HashAlgorithm);
```

Comment:
Let's put the s_defaultAlgorithm back, so it's easier to manage, but it can just use the new cached instance.